translations/messages.nl.yaml

```diff
@@ -0,0 +1,7 @@
+readid.authorization.title: 'Identiteitscontrole'
+readid.authorization.text: 'Onherstelbare error'
```

Comment:
Maybe 'fout' is more mainstream (and iirc is used in the other openconext applications)